qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/UnwrappedSurface.h

```diff
@@ -176,6 +182,10 @@ protected slots:
 
   /// Zoom stack
   QStack<RectF> m_zoomStack;
+
+  QSize m_widgetSize;
+
+  bool m_correctAspectRatio; ///< whether to maintain aspect ratio if widget has different aspect ratio to the data
```

Comment:
In short - yes I think I should change the boolean flag to m_maintainAspectRatio.
There's a function Viewport::correctForAspectRatioAndZoom that some of my code changes here are based on eg I've created a variant of this function in UnwrappedSurface.cpp. So when I created a boolean property to control whether this function gets called I ended up using similar language. It would have been one step better if I had included the word "for" while doing this ie m_correctForAspectRatio. However this is a bit of an internally focussed name - it's an implementation detail that we need to work hard to correct the plot area to maintain the aspect ratio. So prefer the "Maintain aspect ratio" name for the flag. I suggest I keep the function name as it is though to emphasise the similarity with the function in Viewport.pas.